### ISSUE TITLE
Updating Repositories and upgrading installed packages FAILED

### ISSUE THREAD

**Issue by Alluuu** [openhab/openhabian#406]
Along with many other Raspberry Pi 3 B+ issues, out of which many have been attended to, but not yet on a released version of OpenHABian - there seems to be another issue that arises once the WiFi unblocking has been dealt with:

Even though the repository http://raspbian.raspberrypi.org/repository/raspbian is live, the Updating and upgrading of the initial setup fails, for this you need to reinstate dpkg:

`sudo dpkg --configure -a`

For a solution I propose, in first-boot.sh #405

My wifi issue was also only solved after following #387 

Best,

Alari

**Comment by EliasGabrielsson** [comment]
As I understood it this was fixed in https://github.com/openhab/openhabian/pull/405 so I close this now. Correct me if I am wrong and reopen this.
